src/main.cpp

```diff
@@ -115,748 +127,737 @@ int main ( int argc, char** argv )
     QString      strClientName               = "";
     QString      strJsonRpcSecretFileName    = "";
 
-#if !defined( HEADLESS ) && defined( _WIN32 )
-    if ( AttachConsole ( ATTACH_PARENT_PROCESS ) )
-    {
-        freopen ( "CONOUT$", "w", stdout );
-        freopen ( "CONOUT$", "w", stderr );
-    }
-#endif
-
-    // When adding new options, follow the same order as --help output
-
-    // QT docu: argv()[0] is the program name, argv()[1] is the first
-    // argument and argv()[argc()-1] is the last argument.
-    // Start with first argument, therefore "i = 1"
-    for ( int i = 1; i < argc; i++ )
+    try
     {
 
```

Comment:
Sorry, but because of moving this try, the indenting changed an a large part of the code and now the diff looks very confusing...

Reply:
There's a "Hide whitespace" option for Github, and for git there is [`git diff -w`](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt--w)

> `-w`
> `--ignore-all-space`
>
>    Ignore whitespace when comparing lines. This ignores differences even if one line has whitespace where the other line has none.


![image](https://user-images.githubusercontent.com/287742/166106118-1a959b9f-7785-4436-a64f-8e51d1605a54.png)